store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala

```diff
@@ -364,7 +364,14 @@ class PGSecondary(val config: Config) extends Secondary[SoQLType, SoQLValue] wit
             case Some(ci) =>
               CopyDroppedHandler(pgu, ci)
             case None =>
-              throw new Exception("cannot find copy to drop")
+              // ok, we can't find the exception to drop.  This is probably down
```

Comment:
Need to figure out if we update this comment to reflect this actually being a temporary workaround for the other bug or figure out another approach to making the data right.